src/client/src/apps/MainRoute/routes/ShellRoute.tsx

```diff
@@ -33,6 +35,13 @@ const ShellRoute: React.FC<Props> = ({ header }) => {
   const blotter = useSelector(blotterSelector)
   const analytics = useSelector(analyticsSelector)
 
+  useEffect(() => {
```

Comment:
Openfin looks for container with `id="layout-container` on `DOMContentLoaded` to load the platform views. React needs to finish loading and rendering before Openfin can look for this container, otherwise nothing loads. To get around this timing issue I manually call `DOMContentLoaded`. 